src/Analyzer/QueryTreeBuilder.cpp

```diff
@@ -1071,4 +1087,42 @@ QueryTreeNodePtr buildQueryTree(ASTPtr query, ContextPtr context)
     return builder.getQueryTreeNode();
 }
 
+QueryTreeNodePtr QueryTreeBuilder::setFirstArgumentAsParameter(const ASTFunction * function, const ContextPtr & context) const
```

Comment:
Could you reflect in a function name that it is called for groupConcat, please?

Reply:
Actually, I named it this way, considering that we may have similar cases in the future and could reuse this function to do the same job. But if it's wrong in your opinion, I can clarify the `groupConcat` in the function name

Reply:
I can't say it is wrong, this is the question of taste actually :) We can leave it as it is